plans/add_compiler.pp

```diff
@@ -7,20 +7,37 @@
 # @param primary_host _ The hostname and certname of the primary Puppet server
 # @param primary_postgresql_host _ The hostname and certname of the PE-PostgreSQL server with availability group $avail_group_letter
 plan peadm::add_compiler(
-  Enum['A', 'B'] $avail_group_letter,
+  Optional[Enum['A', 'B']] $avail_group_letter = 'A' ,
```

Comment:
I think `Optional` doesn't work when the variable has a default value other than `undef`. In this case you can omit `Optional` and just use `Enum`

Reply:
Seemed to work either way, but optional isn't necessary so removing it. Thanks!